scripts/file_resume.js

```diff
@@ -72,10 +72,7 @@ $(function() {
             // a file has been selected
             resumable.upload();
         } else {
-            // no file selected, the resumable submit button will do nothing
-            // so submit the old form in 'upload' mode, with no file
-            showProgress(uploadMessages.working);
-            $("#upload_form").submit();
+            alert(uploadMessages.noFile);
```

Comment:
For returning a project to PP pool it is ok to not upload a file so we need to allow for this.
If there should be a file this will get caught at the server afterwards.

Reply:
Yes, thanks. Sharon caught this too and is testing a fix that accounts for this.

Reply:
New code update adds a second submit button for the forms that can be submitted without a file. This ensures that all file uploads actually come with uploads.